examples/tests/valid/capture_containers.w

```diff
@@ -18,7 +18,7 @@ let handler = inflight (s: str): str => {
   assert(my_map.size() == 2);
 
   assert(arr_of_map.at(0).has("bang"));
-  assert("world" == j.get("b"));
+  assert(j.get("b") == "world");
```

Comment:
Is this just a stylistic change?

Let's add a check that both should work:
```suggestion
  assert(j.get("b") == "world");
  assert("world" == j.get("b"));
```

Reply:
It's not a stylistic change see #1717. I discussed with @hasanaburayyan and for now this switch is a workaround to get the tests to pass until we handle the issue.